src/main/java/io/excaliburfrc/robot/subsystems/Intake.java

```diff
@@ -52,48 +51,57 @@ public Intake() {
         intakeMotor.setPeriodicFramePeriod(PeriodicFrame.kStatus1, MAXIMAL_FRAME_PERIOD),
         intakeMotor.setPeriodicFramePeriod(PeriodicFrame.kStatus2, MAXIMAL_FRAME_PERIOD));
 
-    automaticCommand =
-        new ConditionalCommand(
-            // increment ball count; input until upper sensor detects a ball
-            new FunctionalCommand(
-                // init
-                ballCount::incrementAndGet,
-                // exe
-                () -> {
-                  intakeMotor.set(Speeds.intakeInDutyCycle);
-                  upperMotor.set(Speeds.upperInDutyCycle);
-                },
-                // end
-                _interrupted -> {
-                  intakeMotor.set(0);
-                  upperMotor.set(0);
-                },
-                // isFinished
-                upperBallTrigger,
-                this),
-
-            // output sets motor until ball entry sensor no longer sees a ball
-            new FunctionalCommand(
-                // init
-                () -> {},
-                // exe
-                () -> intakeMotor.set(Speeds.ejectDutyCycle),
-                // end
-                _interrupted -> intakeMotor.set(0),
-                // isFinished
-                intakeBallTrigger.negate()),
-            // decides by ball color
-            this::isOurColor);
-    // schedule the command whenever the entry sensor newly activates ...
-    intakeBallTrigger.whenActive(automaticCommand);
-
     // update the counter whenever we shoot a ball
     upperBallTrigger.whenInactive(ballCount::decrementAndGet, this);
     // and report if we pass the limit
     new Trigger(() -> ballCount.get() > MAX_BALLS)
         .whenActive(() -> DriverStation.reportWarning("Too many Cargo on robot!", false));
   }
 
+  public Command automaticCommand() {
```

Comment:
Rename this to `intakeBallCommand` or something that better describes what the command actually does?